src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -252,6 +252,11 @@ def save_registration_data(
                             )
                         )
 
+    def build_objecttype_url(self, options: OptionsT) -> str:
+        base_url = options["objects_api_group"].objecttypes_service.api_root
+
+        return f"{base_url}objecttypes/{options['objecttype']}"
```

Comment:
I don't really like the constraint of having to pass a whole `object_type` dict to be able to create an object, can't we just pass the objecttype uuid?

Reply:
the objects API client does not have the necessary information to build the URL to the objecttypes API

Reply:
Sorry I meant the objecttype url

Reply:
ah yeah, also works for me :) implementation detail

Reply:
I updated the logic to handle it this way. Lots of tests are updated because I had to add an extra mocked request in v1 tests (because we now hit the objecttypes API during registration).

However, there's an annoying issue with v2 (docker) tests:
- The configured test objecttypes service points to `http://localhost:8001/api/v2/`, and this works when fetching the objecttype URL during registration.
- But the objects API instance has a reference to `http://objecttypes-web:8000/api/v2/"`, as it communicates through the Docker network

Reply:
you mean in the response data of the objects API, so that making asserts is a bit weird? or are you actually getting network errors?

Reply:
Hrm, so the URL that enters objects-api is `http://localhost:8001/api/v2/objecttypes`, which objects API tries to resolve but can't because the service is defined as `http://objecttypes-web:8000/api/v2/` instead.

I was looking if we can leverage `host.docker.internal`, but that doesn't seem a solution either because it requires us to edit `/etc/hosts` on the host system and that's just too annoying.

However - I think we can do some reverse-proxy trickery in docker-compose - if we put nginx in frontend of object types and objects API, we can make that rewrite `localhost:8000` and `localhost:8001` URLs to a `Host` header of `http://objects-web:8000` and `http://objecttypes-web:8000`. So we can still poke in from outside through localhost, and we still reach the URLs inside.